src/app/request-investigation/request-investigation.component.html

```diff
@@ -1,24 +1,20 @@
 <div class="header">
-  <h1 class="title">Investigative Support</h1>
+  <h1 class="title">Unmask the trolls</h1>
   <p class="subtitle">
-    Let our experts uncover the trolls behind your online harassment.
+    How our investigators can help
```

Comment:
**How our investigators can help** or _How can our investigators help?_

Reply:
Perhaps it is a typo, but that is what is in the requested content change

Reply:
Cool, please confirm.

Reply:
@kilemensi Confirmed this. It is correct as is.